.gitignore

```diff
@@ -21,3 +21,4 @@ test_uwsgi_failed
 .idea
 .pytest_cache/
 venv/
+.vscode
```

Comment:
Is this intentional?

Reply:
While people could just use `.git/info/exclude` for local blacklisting of files, I think it makes sense since `.idea` is already in gitignore as well. (but I'd move this line next to the .idea one to keep IDE/editor-related blacklists together)

Reply:
File removed from change, may be introduced in different PR.